pkg/rancher-desktop/utils/__tests__/paths.spec.ts

```diff
@@ -78,6 +78,16 @@ describe('paths', () => {
       linux:  RESOURCES_PATH,
       darwin: RESOURCES_PATH,
     },
+    deploymentProfileSystem: {
+      win32:  '',
```

Comment:
I would throw exceptions for these, so I'd make this line and the next `win32` line:
```
win32:  new Error('Windows profiles will be read from Registry'),
```

Reply:
New error added.